src/uwtools/resources/jsonschema/chgres-cube.jsonschema

```diff
@@ -27,12 +27,57 @@
             "update_values": {
               "properties": {
                 "config": {
-                  "additionalProperties": {
-                    "type": [
-                      "array",
-                      "boolean",
-                      "number",
-                      "string"
+                  "additionalProperties": false,
+                  "anyOf": [
```

Comment:
Given the many different combinations, I don't think we want to apply these types of requirements. I hope that the dependencies below are sufficient. 

If any of the special "files" entries exist, the corresponding "dir" entry must also be there.

Reply:
While I like the dependencies, thats a fair point - I think it should cover everything this way and consistently with the other schemas

Reply:
In the chgres_cube code, though, it's not straightforward about which combinations of directories and files are required together. For example, if you run with grib2 input you need one set with optional additional files, and that's totally different than nemsio.

I'd prefer to stay out of that logic with our schema and to just indicate that if the user provides the files, then there is the appropriate directory also provided.